generators/entity-client/templates/angular/src/main/webapp/app/entities/pagination-template.ejs

```diff
@@ -18,6 +18,7 @@
 -%>
     <%= entityInstancePlural %>?: I<%= entityAngularName %>[];
     eventSubscriber?: Subscription;
+    queryParamMapObs?: Subscription;
```

Comment:
not needed since unsubscriptions from route observables are already managed

Reply:
✅